dask_ml/model_selection/_split.py

```diff
@@ -424,9 +424,9 @@ def train_test_split(
         test_size = 0.1
 
     if train_size is None and test_size is not None:
-        train_size = 1 - test_size
+        train_size = round(1 - test_size, 6)
```

Comment:
Perhaps, the number 6 could be promoted to a named variable to avoid copy-and-paste bugs upon possible updates to this.

Also, I think for some users, the value 6 wouldn't be as effective, as it would for others. Perhaps, this warrants a consideration for introducing a configurable parameter. If such parameter is absent, some default value (eg, 6, as it has been the choice here) could be used as a fallback.